frontend/lib/src/components/widgets/FileUploader/UploadedFile.tsx

```diff
@@ -56,20 +56,7 @@ export const UploadedFileStatus = ({
   fileInfo,
 }: UploadedFileStatusProps): React.ReactElement | null => {
   if (fileInfo.status.type === "uploading") {
-    return (
-      <ProgressBar
-        value={fileInfo.status.progress}
-        size={Size.SMALL}
-        overrides={{
-          Bar: {
-            style: {
-              marginLeft: 0,
-              marginTop: "4px",
```

Comment:
This margin does not seem to have any effect on the progress bar in file uploader